### ISSUE TITLE
Menu - Accounts visible or invisible

### ISSUE THREAD

**Issue by fstcapb** [eikek/sharry#239]
When I use the register button to create an internal account, the account gets an admin flag of false in the accounts_ table in the database and the account entry in the menu is not visible. If I set the admin flag to true using sql, the accounts menu is visible and the user is able to act with admin privileges. 

If I register via command module (I use it for ldap authentication but of one could also test it with /bin/true e.g. ) the account menu is visible and every user can act as admin - doesn't matter if he has admin=true flag or not.
Can I influence the account menu to be visible / invisible for extern accounts? I do not want all ldap users to be able to act as admins :)

**Comment by eikek** [comment]
To the first part: yes, this is how it is intended currently. I know it's a bit tedious at times. One could set for example a fixed admin account to be able to login as admin from time to time to make changes to accounts.

Second part: Oh this sounds scary. Do you mean that your external accounts are always recognized as admins? 

I'm sorry, I think I don't understand the last question: If external accounts are recognized as admins, this is a huge bug then. If not, would you want to make the account page visible to external accounts?

**Comment by fstcapb** [comment]
Hi eikek,

yes, external accounts have always admin rights in my setting. It doesn't matter if they have admin=true flag or not and I didn't find a way yet to switch them to normal users.

The account page is always visible to external accounts. This is how they can do admin stuff.

**Comment by eikek** [comment]
Hi @fstcapb , oh no… this is really scary and certainly not intended! I tried yesterday with external accounts using OAuth and there I couldn't see this issue. I'm going to take a deeper look now. Which login module do you use?

**Comment by eikek** [comment]
Never mind, I found it! How aweful! I'm going to prepare a new release. Thank you very much for reporting!

**Comment by fstcapb** [comment]
Hi Eike,

thank you very much for the new release with the new features. Snakeoil integrations works good, I think.

Unfortunately it also looks like the bug discussed here is open again. 

Best, Falk

**Comment by eikek** [comment]
Oh my… how did I manage to do this. Which external auth method did you use? (command, http, …)

**Comment by fstcapb** [comment]
I use command as auth method.

**Comment by eikek** [comment]
Hi @fstcapb , I just tried to reproduce this, but couldn't. Here is what I did
- enabled the command auth module and pointed it to a bash script that always returns 0
- logged in with some username and password, which was picked up
  ```
  DEBUG s.b.a.CommandAuth - CommandAuth: starting login UserPassData(user3,***)
  DEBUG s.b.a.C.RunCommand - Result of external auth command: Right(0)
  ```
- In the app there is no Accounts menu available. The db has the admin flag set to `false`

Did I overlooked something? Or how did you trigger this?

**Comment by fstcapb** [comment]
I am also not able to reproduce that error. I'm very sorry. 
I installed sharry 1.5.0 over 1.4.3, stopped the service, dropped the database, changed the sharry config settings, recreated the database and started sharry again. After that I could access the account menu and set admin privileges to my account. It maybe was caused by browser cache or something like that. I cannot reproduce that on the same system with a second account nor can I reproduce that on a second machine with the same account.
